docs/api/v1.24.md

```diff
@@ -4576,7 +4580,11 @@ image](#create-an-image) section for more details.
             "max-size": "10M"
           }
         },
-        "Placement": {},
+        "Placement": {
+          "Constraints": [
+            "node.role == worker"
+          ]
+        }
```

Comment:
oh, good catch, I checked the first one, but missed that the second one _did_ need a comma 😅